packages/frontend/editor-ui/src/components/RunData.vue

```diff
@@ -1548,7 +1645,42 @@ defineExpose({ enterEditMode });
 				</slot>
 			</N8nCallout>
 		</div>
-
+		<div
+			v-if="getConsolidatedHint() && props.paneType === 'output'"
+			:class="[$style.hintCallout, $style.customHint]"
+		>
```

Comment:
RunData.vue is already a huge file, could you pull these warnings into a new component?